components/Navbar/desktop/index.tsx

```diff
@@ -0,0 +1,33 @@
+'use client';
+
+import ThemeSwitch from '@/components/Theme';
+import NavMenuItems from '../NavMenuItems';
+import DownloadIcon from '../mobile/DownloadIcon';
+
+const DesktopNav = () => {
+  return (
+    <nav className='hidden w-full sm:flex'>
+      <div className='flex-between items-center gap-6'>
+        <div className='flex gap-6'>
+          <NavMenuItems />
+        </div>
+        <div className='flex items-center gap-6 hover:cursor-pointer'>
```

Comment:
```suggestion
        <div className='flex items-center gap-6 select-none'>
```
Let's remove this pointer for the whole div. We end up with a pointer even when the mouse is on the divider. `select-none` makes it so users won't be able to highlight the divider.